Gemfile

```
@@ -15,6 +15,10 @@
 gem "vcr"
 gem "webmock"
 
+gem "faraday"
+
+gem 'json', '~> 2.8'
```

RuboCop (1.75.5)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)